cluster/images/hyperkube/master.json

```diff
@@ -23,6 +23,7 @@
               "apiserver",
               "--portal-net=10.0.0.1/24",
               "--address=127.0.0.1",
+              "--read-write-port=8080",
```

Comment:
please don't add new things that depend on an unsecured port or we'll never get #13598 and #12968 fixed